images/Dockerfile.cache-server

```diff
@@ -0,0 +1,6 @@
+# Use distroless as minimal base image to package the manager binary
```

Comment:
Should we invert the name of these files so we keep syntax highlighting and such? e.g. `cache-server.Dockerfile?

Reply:
We could, or perhaps a subdirectory would be clearer e.g `images/cache-server/Dockerfile`?